amundsenatlastypes/kickstart.py

```diff
@@ -19,11 +27,30 @@ def apply_qn_regex(name, column_qn_regex):
             return column_qn_regex.match(name)
 
         _regex_result = apply_qn_regex(qualified_name, qn_regex)
+
+        if not _regex_result:
+            qn_regex = re.compile(r"""
```

Comment:
Probably re.compile(...) can be also inside apply_qn_regex and then you can compose alternatives as apply_qn_regex(...) or apply_qn_regex(...) or apply_qn_regex(...)

Reply:
Nice!! I already have a TODO here, which means I need to put this code in `pyatlaslclient`. Will do that in that package.